Code/GraphMol/MolDraw2D/MolDraw2D.h

```diff
@@ -449,6 +449,10 @@ class RDKIT_MOLDRAW2D_EXPORT MolDraw2D {
     return atom_syms_[activeMolIdx_];
   };
 
+  virtual void drawArrow(const Point2D &cds1, const Point2D &cds2,
+                         bool asPolygon = false, double frac = 0.05,
```

Comment:
Does the "asPolygon" control whether it is filled or not?  What are the use cases for polygon or lines (I'm assuming?)

Reply:
@greglandrum Just a clarification here, I'm happy with the PR.

Reply:
eh, yeah, sorry. I made the changes locally and forgot to push them.